documentation/cobigen-intellij-installation.asciidoc

```diff
@@ -0,0 +1,62 @@
+:toc:
+toc::[]
+
+
+= Installation
+
+
+== IntelliJ setup
+
+If you want to use IntelliJ for CobiGen project, you can follow the steps below:
+
+. You can install and create script for IntelliJ as described in
+
+`https://github.com/devonfw/ide/blob/master/documentation/intellij.asciidoc`
+
+. After starting IntelliJ, you can import CobiGen as maven project
```

Comment:
Needs to be emphasized.
You **need** to import the cobigen folder as a maven project.